app/uk/gov/hmrc/cdsimportsddsfrontend/views/components/documentation_type.scala.html

```diff
@@ -0,0 +1,38 @@
+@*
```

Comment:
Similar to above, 'documentation.scala.html' or whatever is in Customs_Declaration_Service_-_UK_Trade_Tariff_guide.pdf